hw3/index.html

```diff
@@ -0,0 +1,193 @@
+						ultricies lacinia pulvinar
+					</div>
+					<div class="preview__text">
+						Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.
+					</div>
+				</div>
+				<button class="preview__button">See more</button>
+			</section>
+			<section class="preview__item preview__item-top">
+				<div class="preview__container">
+					<img src="image/icon-2.png" class="preview__logo">
+					<div class="preview__title">
+						magnust minima sed quia
+					</div>
+					<div class="preview__text">
+						Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea asekin morir commodi consequatur?
+					</div>
+				</div>
+				<button class="preview__button">See more</button>
+			</section>
+			<section class="preview__item preview__item-top">
+				<div class="preview__container">
+					<img src="image/icon-3.png" class="preview__logo">
+					<div class="preview__title">
+						ultricies lacinia pulvinar
+					</div>
+					<div class="preview__text">
+						Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. 
+					</div>
+				</div>
+				<button class="preview__button">See more</button>
+			</section>
+			<section class="preview__item">
+				<div class="preview__picture preview__picture-1">
+					<a href="#" class="preview__link">a</a>
+				</div>
+				<div class="preview__title">
+					magnust alecet fernum
+				</div>
+				<div class="preview__text">
+					Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et.
+				</div>
+			</section>
+			<section class="preview__item">
+				<div class="preview__picture preview__picture-2">
+					<a href="#" class="preview__link">a</a>
+				</div>
+				<div class="preview__title">
+					quia porro venium
+				</div>
+				<div class="preview__text">
+					Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et.
+				</div>
+			</section>
+			<section class="preview__item">
+				<div class="preview__picture preview__picture-3">
+					<a href="#" class="preview__link">a</a>
+				</div>
+				<div class="preview__title">
+					dolorest nuwuam et lobiare
+				</div>
+				<div class="preview__text">
+					Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et.
+				</div>
+			</section>
+		</div>
+	</div>
+	<footer class="main-footer">
+		<div class="main-footer__container">
+			<section class="main-footer__item">
+				<div class="main-footer__title">about us</div>
```

Comment:
you see, it is not really `main-footer__title` right? 
It is like main-footer-item__title? 
So you can use section like a block main-footer-item